autograder/submitty_autograding_shipper.py

```diff
@@ -750,6 +751,17 @@ def checkout_vcs_repo(config, my_file, which_branch):
     checkout_path = os.path.join(course_dir, "checkout", partial_path)
     results_path = os.path.join(course_dir, "results", partial_path)
 
+    # Load the git branch for autgrading from the course config file
+    which_branch = 'undefined'
```

Comment:
Might as well set this to what we assume people will want as the default, especially in the case that `git_autograding_branch` is not set and we probably do not want `undefined` as our fallback:
```suggestion
    which_branch = 'main'
```